README.md

```diff
@@ -71,6 +89,7 @@ MSALInteractiveTokenParameters *interactiveParams = [[MSALInteractiveTokenParame
 The master branch has been copied over to main branch. The master branch will contain updates only until version 1.2.14, for further releases please refer to 'main' branch instead of 'master'.
 
 ## Installation
+
 ### Using CocoaPods
```

Comment:
specifically, to native auth, we have different instructions about how to use CocoaPods. Can we create a dedicated CocoaPod section or add a note for native Auth?

Reply:
Great call out, I'll create a different section for this. Thanks for letting me know.